docs/source/getting-started.rst

```diff
@@ -157,24 +175,19 @@ To change what input is fed through the pipeline, modify ``batch_runner.py`` and
     logging.basicConfig(level=logging.INFO)
 
     class BatchRunner(Runner):
-        def run(self, is_training=False):
-            self.assembler.init_assembler(True)
-            data = AssemblyState()
+        def load_data(self, mode, config):
```

Comment:
This could be simplified by adding a constructor to the `AssemblyState` class and makes it clear what the input variables will be (it won't always be a single field). The class should probably be called `FileSystemRunner`.

For example:
```
class FileSystemRunner(Runner):

    def load_data(self, mode, config):
        raw_data = "TODO: Load raw data here"
        return AssemblerState(raw_data)
```